common/web/types/src/lexical-model-types.ts

```diff
@@ -531,14 +531,28 @@ export interface Configuration {
   rightContextCodeUnits?: number,
 
   /**
-   * Whether or not the model appends characters to Suggestions for
-   * wordbreaking purposes.  (These characters need not be whitespace
-   * or actual wordbreak characters.)
+   * Specifies behaviors related to transforms that the active model appends
+   * to Suggestions for wordbreaking purposes.  (The Transforms need not apply
+   * whitespace or actual wordbreak characters.)
    *
    * If not specified, this will be auto-detected based on the model's
-   * punctuation properties (if they exist).
+   * punctuation properties (if they exist).  If left null/undefined, the model
+   * does not append wordbreaking transforms to Suggestions.
    */
-  wordbreaksAfterSuggestions?: boolean
+  appendsWordbreaks?: {
+    /**
+     * Specifies strings that, when input, always act as word-boundaries on the
+     * input - both when typed after a manually-applied suggestion (replacing
+     * appended whitespace) and when typed with an auto-selected suggestion
+     * available (thus accepting it directly, as with whitespace).
+     *
+     * This is designed to allow language-appropriate punctuation marks to
+     * automatically remove whitespace (or other wordbreak characters) as
+     * appropriate, and to auto-accept for inputs that clearly signal intent
+     * to end the current word, both in order to improve user UX with autocorrect.
+     */
+    breakingMarks?: string[];
+  }
```

Comment:
I could not find anywhere without our existing model-templates that actually set this directly.  The old form is currently set in the main `LMLayerWorker` class, after the model's run its configure method, and not before.

As it sought to model what is now just a _subset_ of the new feature, it seemed best to rework this configuration parameter to better serve the new feature's needs.  I understand if we want to workshop the exact specification patterns further before merging, though.